core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -1018,6 +1029,12 @@ abstract class Record extends Aggregate {
     }
   }
 
+  /* Tracking variable for deciding Record flipped-ness. */
+  private[chisel3] var _isFlipped: Boolean = false
+
+  /* In the context of Records, isFlipped is assigned true if any of its children are flipped. */
+  override def isFlipped: Boolean = _isFlipped
```

Comment:
should we rename this to "containsAtLeastOneFlippedElement" or something? As it is it sounds like the whole thing is flipped?

Reply:
should isFlipped be private? I dont think we allow users to extend Data themselves

Reply:
oh it is private